docs/3.api/1.composables/use-async-data.md

```diff
@@ -31,6 +31,7 @@ type AsyncDataOptions<DataT> = {
   transform?: (input: DataT) => DataT
   pick?: string[]
   watch?: WatchSource[]
+  getCachedData?: (key: string) => any
```

Comment:
Why is it `any` here but `getCachedData?: (key: string) => DataT` in the TS files 🤔

Reply:
It should be the same 👍

Reply:
Will send a PR ☺️